test/integration/cluster-init/input/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -0,0 +1,44 @@
+presubmits:
+  openshift/release:
+    - agent: kubernetes
+      always_run: true
+      branches:
+        - master
+      cluster: app.ci
+      context: ci/build-farm/build01-dry
+      decorate: true
+      labels:
+        pj-rehearse.openshift.io/can-be-rehearsed: "true"
+      name: pull-ci-openshift-release-master-build01-dry
+      rerun_command: /test build01-dry
+      spec:
+        containers:
+          - args:
+              - --config-dir=clusters/build-clusters/build01
+              - --as=
+              - --KUBECONFIG=/etc/build-farm-credentials/KUBECONFIG
+            command:
+              - applyconfig
+            image: registry.ci.openshift.org/ci/applyconfig:latest
+            imagePullPolicy: Always
+            name: ""
+            resources:
+              requests:
+                cpu: 10m
+            volumeMounts:
+              - mountPath: /etc/build-farm-credentials
+                name: build-farm-credentials
+                readOnly: true
+              - mountPath: /tmp
+                name: tmp
+        serviceAccountName: config-updater
+        volumes:
+          - name: build-farm-credentials
+            secret:
```

Comment:
@hongkailiu please provide the correct (new) item here

Reply:
```
oc get secret -n ci config-updater -o yaml | yq -r '.data|keys[]'
sa.config-updater.app.ci.config
sa.config-updater.arm01.config
sa.config-updater.build01.config
sa.config-updater.build02.config
sa.config-updater.hive.config
sa.config-updater.vsphere.config
```

This secret comes from https://github.com/openshift/release/blob/686b7865a092452735f086169336f16f497a3c76/core-services/ci-secret-bootstrap/_config.yaml#L1842-L1867